app/minutes/test_minutes.py

```diff
@@ -158,7 +158,7 @@ def test_get_minutes_no_user(self):
 
         received = self.socketio.get_received()
         response = received[0]["args"][0]
-        assert response == Response.UserDoesntExist
+        assert response == []
```

Comment:
I'm not sure a good way to make it clear why this is the desired outcome. The only minutes that were added to the database in `setup_method` are private, so an unauthenticated user should get back an empty array. 

In reality there could be public minutes in the database so there _should_ be another test that verifies that something does get returned when public minutes exist in the database. I'm not sure if we want another test for that case, or to alter the `setup_method` function to include a public minute.

Reply:
Good catch. It may be beneficial to add one public minute in and then modify the test to check for the one public. I think that is more in line with the actual functionality of the site.